File: corp-watch-api/CorpWatchApi/Models/Employee.cs
using System.Collections.Generic;

namespace CorpWatchApi.Models
{
    public class Employee
    {
       
        public long Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public ICollection<Job> Jobs { get; set; }
        public string Department { get; set; }
    }
}

File: corp-watch-ui/src/components/Employee.js
import React, { Component } from 'react';
import AuthService from './AuthService';

class Employee extends Component {
    constructor() {
        super();
        this.state = {
            Employees: []
        };
        this.auth = new AuthService();
    }

    componentWillMount() {
        /*this.auth.*/fetch('http://localhost:5000/api/Employee')
            .then(results => {
                return results.json();
            })
            .then(data => {
                this.setState({
                    Employees: data
                });
            })
    }

    render() {
        return (
            <div className="container1" >
                {this.state.Employees.map(e => (
                    <div>
                        <h3>{e.firstName} {e.lastName}</h3>
                        <p>Jobs: {e.jobs}</p>
                        <p>Department: {e.department}</p>
                    </div>
                ))}
            </div>
        )
    }
}

export default Employee;